vignettes/details.Rnw

```diff
@@ -16,6 +16,7 @@
 
 
 \begin{document}
+\SweaveOpts{concordance=TRUE}
 
```

Comment:
@kshirley [at the top of this document](https://github.com/cpsievert/LDAvis/blob/quick-fix/vignettes/details.Rnw#L1-L2), we tell the R installer to [use knitr when compiling the vignette](http://yihui.name/knitr/demo/vignette/), which isn't directly compatible with Sweave. 

If you want, I could switch from Rnw to Rmd, so we won't have to fiddle with this anymore...